manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Start | Stop",
   "description": "Assign or un-assign yourself from an issue.",
-  "ubiquity:listeners": ["issue_comment.created", "issues.assigned"],
+  "ubiquity:listeners": ["issue_comment.created", "issues.assigned", "pull_request.opened", "pull_request.reopened"],
```

Comment:
Reopened it doesn't seem to make sense

Reply:
I can remove it, I thought it would make sense if a PR is reopened to re-assign the user if it was not assigned anymore.